### PR TITLE
Added a set of helpful cheat/debug commands and bots

### PR DESCRIPTION
**Pull request by darklordabc**
These can be used by any player typing them in chat. They can only be used if sv_cheats is 1 or if the game is in tools mode. 

Currently added: -allvision, -normalvision, -god, -disarm, -dagger, -core 1-4 -startduel

### CODE REVIEW

**Comment by warpdragon** [comment]
Did u push to a closed repo, perhaps?

**Comment by darklordabc** [comment]
No I pushed to my fork of this repo. Are you trying to add to it?

**Comment by warpdragon** [comment]
The PR was closed b4 ur last commit, no?

File: game/resource/English/tooltip_devDagger.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+//====================================================================================
+// Global Range Blink Dagger for Developers to help in testing
+//====================================================================================
+		"DOTA_Tooltip_ability_item_devDagger" "Dev Dagger"
+   	    "DOTA_Tooltip_ability_item_devDagger_Description" "Dagger used by developers to help in testing."
```
Comment (Chronophylos) [inline game/resource/English/tooltip_devDagger.txt]: Here is a missing newline.

**Comment by darklordabc** [comment]
Added -endduel command and -addbots

File: game/scripts/vscripts/events.lua [hunk 2]
```diff
@@ -474,28 +475,154 @@ end
...
+           	elseif level == 2 then
+           		hero:AddItemByName("item_upgrade_core_2")
+           	elseif level == 3 then
+           		hero:AddItemByName("item_upgrade_core_3")
+           	elseif level == 4 then
+           		hero:AddItemByName("item_upgrade_core_4")
+           	end
+
+        end        
+	end
```
Comment (chrisinajar) [inline game/scripts/vscripts/events.lua]: why not use the existing chat command API? It's way more maintainable.

File: game/scripts/vscripts/events.lua [hunk 3]
```diff
@@ -1,5 +1,6 @@
 -- This file contains all barebones-registered events and has already set up the passed-in parameters for your use.
 GameEvents = GameEvents or {}
+require('libraries/timers')
```
Comment (chrisinajar) [inline game/scripts/vscripts/events.lua]: this should already have been included in gamemode

**Comment by darklordabc** [comment]
What do you mean by existing chat command registry?

File: game/scripts/vscripts/events.lua (hunk 2 above)
Comment (darklordabc) [inline game/scripts/vscripts/events.lua]: Moved them to the chat command library.

File: game/scripts/vscripts/events.lua (hunk 3 above)
Comment (darklordabc) [inline game/scripts/vscripts/events.lua]: Removed it

**Comment by chrisinajar** [comment]
still a couple whitespace errors, and imma refactor that code to use the register command method, but this good for now so people can test against it :+1:

**Pull request merged**